homeassistant/components/netatmo/config_flow.py

```diff
@@ -101,8 +102,7 @@ class NetatmoOptionsFlowHandler(OptionsFlow):
 
     def __init__(self, config_entry: ConfigEntry) -> None:
         """Initialize Netatmo options flow."""
-        self.config_entry = config_entry
-        self.options = dict(config_entry.options)
+        self._options = deepcopy(dict(config_entry.options))
```

Comment:
Wait for #129754 / #129756
```suggestion
        self.initialize_options(config_entry)
```